app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,192 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def confirm_sync(params)
+      job = SyncPlaybooks.perform_later(params[:changed], @ansible_proxy)
+      task = ForemanTasks::Task.find_by(external_id: job.provider_job_id)
+      task
+    end
+
+    def import_playbooks(playbooks_names)
+      filter_playbooks = playbooks_names.empty? ? playbooks : find_playbooks(playbooks_names)
+      import_playbooks_content filter_playbooks
+    end
+
+    def import_playbooks_content(playbooks)
+      params = { changed: { 'update': [], 'create': [], 'delete': [] } }
```

Comment:
Delete key is never used, and quotes are not needed as there is no space in the key name.